### PR TITLE
Add current and expected JSON with transformer for event edx.user.settings.viewed

### PR DESCRIPTION
**Pull request by HamzaFarooqArbisoft**
**Trello Link:** _https://trello.com/c/71stOE3E/124-user-settings-edxusersettingsviewed_

**Description:** _Add cyrrent and expected jsons with transformer for event edx.user.settings.viewed_

**Checks before merge:**

- [ ] Reviewed
- [ ] Commits squashed

### CODE REVIEW

File: openedx/features/caliper_tracking/tests/current/edx.user.settings.viewed.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (imhassantariq) [inline openedx/features/caliper_tracking/tests/current/edx.user.settings.viewed.json]: Sort file alphabetically

File: openedx/features/caliper_tracking/tests/expected/edx.user.settings.viewed.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (imhassantariq) [inline openedx/features/caliper_tracking/tests/expected/edx.user.settings.viewed.json]: Sort file alphabetically

File: openedx/features/caliper_tracking/tests/current/edx.user.settings.viewed.json (hunk 1 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/current/edx.user.settings.viewed.json]: done

File: openedx/features/caliper_tracking/tests/expected/edx.user.settings.viewed.json (hunk 2 above)
Comment (HamzaFarooqArbisoft) [reply openedx/features/caliper_tracking/tests/expected/edx.user.settings.viewed.json]: done

**Pull request merged**